.github/pytorch-cached-s3-dependencies.yml

```diff
@@ -0,0 +1,22 @@
+# A list of third-party dependencies cached on OSSCI S3.
+# All updates to this config will be automatically applied by "sync-s3-cache" workflow.
+# For more information and documentation, see:
+# https://github.com/pytorch/builder/main/thirdparty_deps/README.md
+# and https://github.com/pytorch/builder/pull/1096
+validation:
+  # Key: allowed S3 bucket name
+  # Value: list of allowed S3 key prefixes
+  ossci-windows:
+    - "test/"
+    - "miniconda/"
```

Comment:
I think we also have cuda and cudnn on Windows. They are used here https://github.com/pytorch/test-infra/blob/main/aws/ami/windows/scripts/Installers/Install-CUDA-Tools.ps1, and I assume we upload them there manually at the moment

Reply:
Yep, I believe that's the case. 
Thanks for the help in locating such places, I'll be adding them to the config (maybe in subsequent PRs to minimize the scope of the changes).